shapely/strtree.py

```diff
@@ -84,8 +84,9 @@ def __init__(self, geoms):
     def _init_tree_handle(self, geoms):
         # GEOS STRtree capacity has to be > 1
         self._tree_handle = lgeos.GEOSSTRtree_create(max(2, len(geoms)))
```

Comment:
Unrelated to this PR, but the argument you pass to `GEOSSTRtree_create` is the capacity of a tree node, not the tree itself! Using `len(geoms)` will give you a tree with a single leaf, i.e. a glorified array of geometries. 10 is a good value to use.

Reply:
That would explain this: https://gis.stackexchange.com/questions/353619/shapely-with-rtree-versus-strtree

Reply:
Wow, awesome catch Dan. I've seen that question before but had no idea what caused it, glad to see there's a reasonable explanation.

Reply:
Fixed in #1070

Reply:
@dbaston thanks a lot for the catch! 
In pygeos, we are luckily already using a fixed default of 5 (https://github.com/pygeos/pygeos/blob/master/pygeos/strtree.py#L54). Is that a good default as well, or would you recommend changing it to 10?

Reply:
When I've experimented with it I get the best performance in the 8-16 neighborhood. GEOS internally defaults to 10 and it's hard to see a reason to change it. I'm not sure why GEOS exposes this to the user in the first place; I've made the same mistake myself. https://github.com/postgis/postgis/commit/1ff9c1470fb2ca32ab19569bb570decc9f62accb